backend/src/main/resources/logback.xml

```diff
@@ -0,0 +1,37 @@
+<configuration>
```

Comment:
Mieliśmy wrzucić przykładową konfigurację, a właściwą dorzucić do .gitignore.

Reply:
Nie wiem dlaczego nie usunęło tego pliku. Będzie trzeba jeszcze raz na to spojrzeć.